relay-config/src/config.rs

```diff
@@ -586,7 +586,7 @@ struct Limits {
     max_replay_message_size: ByteSize,
     /// The maximum number of threads to spawn for CPU and web work, each.
     ///
-    /// The total number of threads spawned will roughly be `2 * max_thread_count + 1`. Defaults to
+    /// The total number of threads spawned will roughly be `2 * max_thread_count`. Defaults to
```

Comment:
Can we add to this comment that Relay will use at most `max_thread_count` CPU?

Reply:
With the current setup this isn't gurantueed right? If the runtime needs more processing power than expected it can go over.